src/graphdb.js

```diff
@@ -147,39 +186,75 @@ function graphDBInferenceSchema (graphbSchema, addMutations) {
         r += '}\n\n';
 
         // input for the node type
-        r += `input ${nodeCase}Input {\n`;
+        let nodeCase2 = replaceCleanseLabel(node.label);
```

Comment:
Why do you need another variable? Can you just use `nodeCase` which should already be handled for special characters at this point?